### ISSUE TITLE
Performance is poor

### ISSUE THREAD

**Issue by KejieLyu** [ildoonet/tf-pose-estimation#336]
Hello, thanks for your codes. 
However, I tried to retrain the vgg based model using the following configurations:
lr = 0.0001, batchsize=64, input-width = input-height = 368x368
And the AP is only 0.364 when step=40000.
![image](https://user-images.githubusercontent.com/33347544/46783933-bdd0d580-cd5e-11e8-8cf2-e51749b45d29.png)

I don't know where is wrong. The training loss is as follows:
![image](https://user-images.githubusercontent.com/33347544/46783964-e35ddf00-cd5e-11e8-9eef-7bc33e6f53d0.png)
![image](https://user-images.githubusercontent.com/33347544/46783977-f375be80-cd5e-11e8-9f2b-122fde27d5ea.png)

**Comment by Sierkinhane** [comment]
have you solved it？ what's your ultimate train loss and val loss?

**Comment by ildoonet** [comment]
please, try with lr=0.001.

I will update the training documents after I run some experiments with mobilenet-v2.

Thanks.

**Comment by KejieLyu** [comment]
@ildoonet  Thanks for your recommendation, I will try that lr.
